Dockerfile

```diff
@@ -16,22 +33,15 @@ RUN apt-get update -y \
 # https://github.com/pytorch/pytorch/issues/107960 -- hopefully
 # this won't be needed for future versions of this docker image
 # or future versions of triton.
-RUN ldconfig /usr/local/cuda-12.4/compat/
+RUN ldconfig /usr/local/cuda-$(echo $CUDA_VERSION | cut -d. -f1,2)/compat/
 
 WORKDIR /workspace
 
 # install build and runtime dependencies
 COPY requirements-common.txt requirements-common.txt
 COPY requirements-cuda.txt requirements-cuda.txt
 RUN --mount=type=cache,target=/root/.cache/pip \
-    pip install -r requirements-cuda.txt
-
-# install development dependencies
-COPY requirements-lint.txt requirements-lint.txt
-COPY requirements-test.txt requirements-test.txt
-COPY requirements-dev.txt requirements-dev.txt
-RUN --mount=type=cache,target=/root/.cache/pip \
-    pip install -r requirements-dev.txt
+    python${PYTHON_VERSION} -m pip install -r requirements-cuda.txt
```

Comment:
if update alternative is called, do we still need this?

Reply:
Ah I just tested again and ya we don't need it, calling `python3` would be sufficient. I updated it only call `python3 -m pip ... ` now and `python3 ...` now.

I think my previous run used the wrong Python version for some reason so that's why I added the version in but it should be okay now.